plugins/event_notification/lib/model/data/kEventNotificationDispatchJobData.php

```diff
@@ -14,14 +14,35 @@ class kEventNotificationDispatchJobData extends kJobData
 	 * @var int
 	 */
 	protected $jobId;
+
+	/**
+	 * @var KalturaEventNotificationDelayedConditions
```

Comment:
I believe here we need the core object of EventNotificationDelayedConditions since we are in a a core class